modules/imap_folders/modules.php

```diff
@@ -576,19 +580,24 @@ function get_sieve_linked_mailbox ($imap_account, $module) {
         require_once VENDOR_PATH.'autoload.php';
         $sieve_options = explode(':', $imap_account['sieve_config_host']);
```

Comment:
Shouldn't we parse this configuration with the new parse_sieve_config_host function?